libvast/src/experimental_table_slice.cpp

```diff
@@ -681,6 +681,27 @@ class record_batch_decoder final {
   std::shared_ptr<arrow::RecordBatch> record_batch_ = nullptr;
 };
 
+/// Compute position for each array by traversing the schema tree breadth-first.
+void index_column_arrays(const std::shared_ptr<arrow::Array>& arr,
+                         arrow::ArrayVector& out) {
+  auto f = detail::overload{[&](const std::shared_ptr<arrow::Array>& a) {
+                              out.push_back(a);
+                            },
+                            [&](const std::shared_ptr<arrow::StructArray>& s) {
+                              for (const auto& child : s->fields())
+                                index_column_arrays(child, out);
+                            }};
```

Comment:
```suggestion
                            },};
```

This makes clang-format handle this one better.

Reply:
> This makes clang-format handle this one better.

next PR removes the entire file, so I'm fixing it in the new place instead.